src/free-style.ts

```diff
@@ -170,10 +170,11 @@ function stringifyProperties (properties: Properties) {
 
   for (const [name, value] of properties) {
     if (value != null) {
-      if (Array.isArray(value)) {
-        result.push(value.filter(x => x != null).map(x => styleToString(name, x)).join(';'))
+      const value2 = typeof value === 'function' ? value() : value
```

Comment:
And what about vendor prefixing of the value? It would be skipped in that case. I don't think it's a worthwhile change to break expectations at this point, that's all.